website/components/navigation/MobileNavigation.tsx

```diff
@@ -71,11 +71,14 @@ function MobileNavigation({ children }: { children: React.ReactNode }) {
   const scrollContainerRef = useRef<HTMLDivElement>(null);
   const [open, setOpen] = useState(false);
 
-  const handleBackdropClick = (e: React.SyntheticEvent) => {
-    if (!scrollContainerRef?.current?.contains(e.target as Node)) {
-      setOpen(false);
-    }
-  };
+  const handleBackdropClick = useCallback(
+    (e: React.MouseEvent) => {
+      if (!scrollContainerRef?.current?.contains(e.target as Node)) {
+        setOpen(false);
+      }
+    },
+    [scrollContainerRef],
```

Comment:
I think this will have issues with React 17. should be avoidable if you assign `scrollContainerRef?.current` to a variable outside of the callback and use that in the dependencies.

Reply:
Done 👍